libs/components/phone-field/src/lib/modules/phone-field/phone-field.component.scss

```diff
@@ -21,47 +42,27 @@
   flex-grow: 1;
 }
 
-.sky-phone-field-search-btn-dismiss {
-  border-color: transparent;
-}
-
-.sky-phone-field-country-selected,
-.sky-phone-field-country-default {
-  @include mixins.sky-border(light, bottom);
-}
-
-.sky-phone-field-search-result {
-  .sky-phone-field-flag {
-    display: inline-block;
-    margin-right: $sky-margin-half;
-  }
-}
-
+// Default only style for the dismiss icon in input boxes
 .sky-input-box {
   .sky-phone-field-search-btn-dismiss .fa-lg {
     vertical-align: -10%;
   }
 }
 
-.sky-theme-modern {
-  .sky-input-box {
-    .sky-phone-field-country-search .sky-country-field-input {
-      background-color: transparent;
-    }
-
-    .sky-phone-field-country-btn.sky-input-group-btn .sky-btn {
-      padding: 0 17px 0 18px;
-    }
+.sky-input-box {
+  .sky-phone-field-country-btn.sky-input-group-btn .sky-btn {
+    padding: var(--sky-override-phone-field-input-box-country-button-padding);
```

Comment:
An ask from design was to use the same method for centering things in the button as what datepicker/timepicker do. The padding here was the old way. Removing the padding in v2 and using the `display: inline-block` allows the flag to be centered in the same way that our icons are.